examples/ERP/plot_embedding_MEG.py

```diff
@@ -1,25 +1,26 @@
 """
-=====================================================================
-Embedding ERP MEG data in 2D Euclidean space with Laplacian Eigenmaps
-=====================================================================
+============================================
+Embedding ERP MEG data in 2D Euclidean space
+============================================
 
-Spectral embedding via Laplacian Eigenmaps of a set of ERP data.
+Embeddings via Laplacian Eigenmaps and Riemannian locally linear
+embedding of a set of ERP data.
 
 """
-# Authors: Pedro Rodrigues <pedro.rodrigues01@gmail.com>
-#
+# Authors:  Pedro Rodrigues <pedro.rodrigues01@gmail.com>,
+#           Gabriel Wagner vom Berg <gabriel@bccn-berlin.de>
 # License: BSD (3-clause)
 
 from pyriemann.estimation import XdawnCovariances
-from pyriemann.embedding import Embedding
+from pyriemann.utils.viz import plot_embedding
```

Comment:
honestly I would not start putting such a simple viz code in the public API only for illustration in one example.

I would keep for now the plot_embedding function defined here.

Reply:
sorry, what?

Reply:
If I understood correcly, @agramfort is saying that the new function that you've created `plot_embedding` should be defined here in the example script and not as an independent one to be called from the public API. I agree with him.

Reply:
Ah, but the function already existed before, I just adapted it so it could plot both embeddings.

Reply:
I agree with @gabelstein : this function has been added in `viz` module in pyRiemann 0.2.6, and can not be deleted like that.